Sources/SnapshotTesting/AssertSnapshot.swift

```diff
@@ -38,8 +38,7 @@ public func assertSnapshot<Value, Format>(
     record: recording,
     timeout: timeout,
     file: file,
-    testName: testName,
-    line: line
```

Comment:
turns out `verifySnapshot` doesn't need `line` since it isn't doing any asserting.

Reply:
Ah nice...this would technically be a breaking change, though, for 
https://github.com/Killectro/swift-snapshot-testing-nimble

Not sure if anyone other than @Killectro has used this API, though!

Reply:
good point, im gonna revert for now, not hurting anything.